scripts/prepare-dfx-assets.sh

```diff
@@ -7,14 +7,14 @@ SDK_ROOT_DIR="$( cd -- "$(dirname -- "$( dirname -- "${BASH_SOURCE[0]}" )" )" &>
 # shellcheck disable=SC1090
 source "$SDK_ROOT_DIR/scripts/dfx-asset-sources.sh"
 
-DFX_ASSETS_FINAL_DIR=${1?'Must specify a destination directory.'}
+DFX_frontend_FINAL_DIR=${1?'Must specify a destination directory.'}
```

Comment:
All changes in this file should be reverted
```suggestion
DFX_ASSETS_FINAL_DIR=${1?'Must specify a destination directory.'}
```